recipes/arccommander/meta.yaml

```
@@ -0,0 +1,36 @@
+{% set name = "arccommander" %}
+{% set version = "0.3.3" %}
+{% set sha256 = "4abcebb9e59537c211f955cf638799d3f832b77b99a0dcf5ba5a80c3376a5f23" %}
+
+package:
+  name: {{ name|lower }}
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/nfdi4plants/arcCommander/archive/refs/tags/Installer_v{{ version }}.tar.gz
+  sha256: '{{ sha256 }}'
+
+build:
+  number: 0
+
+requirements:
+  host:
+    - dotnet-sdk
+  run:
+    - dotnet-runtime
```

As we're using the SDK as a compiler I think it should be in the build section. I'm not fully across the intricacies of cross-compiling with `dotnet` but I *think* we should have the runtime in the host as well as run - e.g.
```suggestion
requirements:
  build:
    - dotnet-sdk
  host:
    - dotnet-runtime
  run:
    - dotnet-runtime
```